doc/source/advanced.rst

```diff
@@ -751,30 +756,31 @@ Scalar selection for ``[],.loc`` will always be label based. An integer will mat
    sf.loc[3]
    sf.loc[3.0]
 
-The only positional indexing is via ``iloc``
+The only positional indexing is via ``iloc``.
 
 .. ipython:: python
 
    sf.iloc[3]
 
-A scalar index that is not found will raise ``KeyError``
+A scalar index that is not found will raise a ``KeyError``.
 
-Slicing is ALWAYS on the values of the index, for ``[],ix,loc`` and ALWAYS positional with ``iloc``
+Slicing is **always** on the values of the index when using ``[],ix,loc``, and 
```

Comment:
Except for booleans :)

Perhaps we can rephrase this as "**primarily** on the values" and mention elsewhere (perhaps after `iloc`) that booleans are also allowed.